client/packages/purchase_orders/src/api/hooks/usePurchaseOrder.ts

```diff
@@ -76,22 +87,37 @@ const useCreate = () => {
   const mutationFn = async (
     input: InsertPurchaseOrderInput
   ): Promise<string> => {
-    const result =
-      (await purchaseOrderApi.insertPurchaseOrder({
-        input,
-        storeId,
-      })) || {};
+    const result = await purchaseOrderApi.insertPurchaseOrder({
+      input,
+      storeId,
+    });
+    if (result?.insertPurchaseOrder.id) return result?.insertPurchaseOrder.id;
+    throw new Error('Could not insert purchase order');
```

Comment:
I'm wondering if these should just return the result - and then handle error notifications if required in the SidePanel component (or wherever you would call the hook)?

Same thing below in the update function